### PR TITLE
Docs Using a Custom Ingestion Image in helm

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Comment by yu-iskw** [comment]
@pmbrull Thank you for writing up the documentation.

File: openmetadata-docs/content/deployment/kubernetes/index.md [hunk 1]
```diff
@@ -237,3 +237,63 @@ global:
...
+USER airflow
+
+# Install our custom connector
+COPY <your_package> <your_package>
+COPY setup.py .
+RUN pip install --no-deps .
+```
+  
+where `openmetadata/ingestion:x.y.z` needs to point to the same version of the OpenMetadata server, for example `openmetadata/ingestion:0.13.1`.
+This image needs to be pushed and available to the container registry of your choice.
```
Comment (akash-jain-10) [inline openmetadata-docs/content/deployment/kubernetes/index.md:266]:
```suggestion
This image needs to be built and published to the container registry of your choice.
```

File: openmetadata-docs/content/deployment/kubernetes/index.md [hunk 2]
```diff
@@ -237,3 +237,63 @@ global:
...
+
+```bash
+helm install openmetadata-dependencies open-metadata/openmetadata-dependencies --values values.deps.yaml
+```
+
+If you had a previous helm installation, you'll need to upgrade via:
+
+```bash
+helm upgrade openmetadata-dependencies open-metadata/openmetadata-dependencies --values values.deps.yaml
+```
```
Comment (akash-jain-10) [inline openmetadata-docs/content/deployment/kubernetes/index.md:295]:
```suggestion
Upgrade/Install your openmetadata-dependencies helm charts with the below single command:

```bash
helm upgrade --install openmetadata-dependencies open-metadata/openmetadata-dependencies --values values.deps.yaml
```

**Review by akash-jain-10 — CHANGES_REQUESTED** [review]

File: openmetadata-docs/content/deployment/kubernetes/index.md [hunk 3]
```diff
@@ -237,3 +237,56 @@ global:
...
+      pullPolicy: "IfNotPresent"
+```
+
+### 3. Install or Upgrade openmetadata-dependencies
+
+Upgrade/Install your openmetadata-dependencies helm charts with the below single command:
+
+```bash
+helm upgrade --install openmetadata-dependencies open-metadata/openmetadata-dependencies --values values.deps.yaml
+
```
Comment (pmbrull) [inline openmetadata-docs/content/deployment/kubernetes/index.md:289]:
```suggestion
```

```

File: openmetadata-docs/content/deployment/kubernetes/index.md [hunk 4]
```diff
@@ -237,3 +237,55 @@ global:
...
+      pullPolicy: "IfNotPresent"
+```
+
+### 3. Install or Upgrade openmetadata-dependencies
+
+Upgrade/Install your openmetadata-dependencies helm charts with the below single command:
+
+```bash
+helm upgrade --install openmetadata-dependencies open-metadata/openmetadata-dependencies --values values.deps.yaml
+### 4. Install OpenMetadata
```
Comment (pmbrull) [inline openmetadata-docs/content/deployment/kubernetes/index.md:289]:
```suggestion
```

### 4. Install OpenMetadata
```

**Review by akash-jain-10 — APPROVED** [review]

**Pull request merged**